doc/lua_api.txt

```diff
@@ -6302,9 +6304,13 @@ Misc.
     * `method` is a string identifying the compression method to be used.
     * Supported compression methods:
         * Deflate (zlib): `"deflate"`
+        * Zstandard: `"zstd"`
     * `...` indicates method-specific arguments. Currently defined arguments
       are:
         * Deflate: `level` - Compression level, `0`-`9` or `nil`.
+        * Zstandard: `level` - Compression level. Integer or `nil`.
```

Comment:
Default compression levels in the `nil` case should be mentioned here.

Reply:
Okay, but if I get another comment telling me to change this *again* (I've received several already bouncing it between various forms, see above) I'm going to start saying this is going too far.
For further information please see: https://github.com/minetest/minetest/pull/12515#issuecomment-1191376302

Reply:
Done as of [b5bdea9](https://github.com/minetest/minetest/pull/12515/commits/b5bdea98418d1c2abc44904e83c8b7ec4ba386b3)

Reply:
Sorry, I TL;DR'd most of it. If it's requested for change again, I'll deal with that.